src/panels/config/script/ha-script-editor.ts

```diff
@@ -446,6 +447,9 @@ export class HaScriptEditor extends LitElement {
         mwc-fab.dirty {
           bottom: 0;
         }
+        .warning {
```

Comment:
No need to include it here, haStyle already has it

Reply:
Thanks for the hint! Commit coming in a few secs.